CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.9.5
```

Comment:
I think that `0.3.10.0` might be a better choice. I know there are not breaking changes. But I think we've been delaying upgrading a version here

Reply:
I totally agree :smile: